app.ts

```diff
@@ -0,0 +1,226 @@
+import { data } from "jquery";
+
+namespace onboardproject {
+	module onboardprojects {
+
+		//Load Variable declarations
+		let Load = 50;
+
+		// Previous Variable declarations
+		let previous: number[];
```

Comment:
Please replace this array with one or two variables. I see you do a lot of sorting on this array and only ever use two the the numbers. You really want to avoid all of that.